packages/shared-components/src/components/Checkbox/Checkbox.js

```diff
@@ -72,6 +72,8 @@ export default class Checkbox extends React.PureComponent {
     Label: CheckboxLabel.Small,
   })(Checkbox);
 
+  defaultId = generateId('checkbox');
```

Comment:
can we move it to `static defaultProps = {  id: generateId('checkbox'), }` ?

Reply:
No, because that will cause all components to share the same id. It needs to be an instance variable.